docs/optimizing/advanced_retrieval/How to get +15% hit_rate improvement using DeepMemory.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>can you mind showing head of train/text examples in the notebook to tell users what is the format like</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
added printings